demo/src/features/text/text_resource.rs

```diff
@@ -89,7 +67,7 @@ impl TextResource {
     }
 
     // Returns the draw data, leaving this object in an empty state
-    pub fn take_text_draw_data(&mut self) -> TextDrawData {
+    pub(in super::super) fn take_text_draw_data(&mut self) -> TextDrawData {
```

Comment:
maybe this should just be pub

Reply:
Replaced `in super::super` with just `super`.